web/controllers/character_controller.ex

```diff
@@ -212,4 +214,18 @@ defmodule EdgeBuilder.CharacterController do
     |> Map.put(:force_power_upgrades, Enum.map(force_power.force_power_upgrades, &ForcePowerUpgrade.changeset/1))
     |> ForcePower.changeset
   end
+
+  defp description_for_character(character, user) do
+    %{username: username} = user
+    %{name: name, species: species, specializations: specializations, career: career} = character
+    "#{name} is #{a_or_an(species)} #{species} #{career} created by #{username} specializing in #{specializations}."
```

Comment:
What's your opinion on adding the character background at the end of this string? About 43% of characters on the site have a background